### ISSUE TITLE
About Magritte

### ISSUE THREAD

**Issue by weslleymberg** [rko281/ReStoreForPharo#2]
Hi. Is there any plan to support [Magritte](https://github.com/magritte-metamodel/magritte) definitions as an alternative to the class side `reStoreDefinition`?

I didn't use your project yet, but I find it simple and interesting. And somewhat the idea of it working together with Magritte seems awesome to me. 

Let me know if it makes sense to you or is just a delusion of mine.

Thanks in advance.

**Comment by rko281** [comment]
Hi there, thanks for the feedback.

The port of ReStore is really my first in-depth work with Pharo so I’m not familiar with Magritte. I’ve taken a quick look and I think there is some overlap like you suggest; I’ll take a deeper look when I have time (or please feel free to do so yourself!)

Best regards,

John Aspinall

**Comment by StephanEggermont** [comment]
Indeed, the reStoredefinition could be easily generated from Magritte definitions. (I'm very familiar with Magritte, not so with ReStore).
